app/src/main/java/de/westnordost/streetcomplete/map/MainFragment.kt

```diff
@@ -566,6 +566,8 @@ class MainFragment : Fragment(R.layout.fragment_main),
         val mapFragment = mapFragment ?: return
         mapFragment.isFollowingPosition = follow
         gpsTrackingButton.isActivated = follow
+        val hideButton = follow && gpsTrackingButton.state !== LocationState.SEARCHING
```

Comment:
> You'll notice that the GPS button's location state is never read from the app logic, only updated.

Fair enough, but s/is/was/, with this implementation.

Reply:
Yes, see https://github.com/westnordost/StreetComplete/pull/2101#pullrequestreview-492221773